### PR TITLE
Redirect rules

### PR DESCRIPTION
**Pull request by fxkopp**
I added SDK support for redirect rules. All new methods have been tested and work fine.

Would be nice if you could merge this, as i wanted to use the package but need support for redirect rules :-)

### CODE REVIEW

**Comment by fxkopp** [comment]
@themsaid Could you check on it? Using it in production for a few weeks now. Works out good.

File: src/Resources/Worker.php [hunk 1]
```diff
@@ -102,17 +102,6 @@ class Worker extends Resource
...
-    /**
-     * Restart the given worker.
-     *
-     * @param  boolean $wait
-     * @return void
-     */
-    public function restart($wait = true)
-    {
-        return $this->forge->restartWorker($this->serverId, $this->siteId, $this->id, $wait);
+        return $this->forge->deleteRedirectRule($this->serverId, $this->siteId, $this->id);
```
Comment (driesvints) [inline src/Resources/Worker.php:105]: @fxkopp why did you change the methods here? This seems off?

Comment (fxkopp) [reply src/Resources/Worker.php:105]: It really seems! Let me take a look tomorrow, made these changes a long time ago.

Comment (driesvints) [reply src/Resources/Worker.php:105]: @themsaid has reverted this: https://github.com/laravel/forge-sdk/commit/d62ecb4b654b0fa5db1dc5e8cb0131bb1ef92d27

Will release tomorrow.

Comment (fxkopp) [reply src/Resources/Worker.php:105]: Thanks! Seems like I had the worker file open in my editor and mixed it up with the Rule file. Sorry again 😬

Comment (fxkopp) [reply src/Resources/Worker.php:105]: @driesvints @themsaid Also fixed the incorrect delete action in the resource class, see #91
